src/components/Snackbar/Snackbar.module.scss

```diff
@@ -0,0 +1,47 @@
+@import '../Animation.scss';
```

Comment:
For the Nora component, we'll basically reimplement this to skin as Tom's designed things